katello/katello/backup.rb

```diff
@@ -0,0 +1,445 @@
+require 'optparse'
```

Comment:
Should we accept these big files now or is this a good time to move them to a repository where there is checking on the ruby code?

Reply:
I would prefer not to go down that path or else this will be blocked for a while. Status quo for the win. It's uglier, I know but as some of us discussed, moving this temporarily to then move it again seems silly when we should just move it to foreman_maintain.

Reply:
It was sort implied that it should move to foreman_maintain but I don't know how difficult that would be.